src/main/app/person/db/queries.sql

```diff
@@ -35,101 +35,101 @@ drop table if exists person_list restrict;
 -- :command :execute
 -- :result :raw
 -- :doc Create person_list_person table
-create table person_list_people (
-    list_id     varchar(255) references person_list (id)
-                  on update restrict
-                  on delete restrict,
-    person_id   int references person (id)
-                  on update restrict
-                  on delete restrict,
-    created_at  timestamp not null default current_timestamp,
-    constraint  person_list_people_pkey primary key (list_id, person_id)
+CREATE TABLE person_list_people (
+    list_id     varchar(255) REFERENCES person_list (id)
+                  ON UPDATE RESTRICT
+                  ON DELETE RESTRICT,
+    person_id   int REFERENCES person (id)
+                  ON UPDATE RESTRICT
+                  ON DELETE RESTRICT,
+    created_at  timestamp NOT NULL DEFAULT current_timestamp,
+    CONSTRAINT  person_list_people_pkey PRIMARY KEY (list_id, person_id)
 );
 
 -- :name drop-person-list-people-table
 -- :command :execute
 -- :result :raw
 -- :doc Drop person_list_person table
-drop table if exists person_list_people restrict;
+DROP TABLE IF EXISTS person_list_people RESTRICT;
 
 -- :name insert-person
 -- :command :execute
 -- :result :affected
 -- :doc Insert person record
-insert into person (name, age)
-values (:name, :age);
+INSERT INTO person (name, age)
```

Comment:
In order to be consistent with other queries (i.e. ['insert-app-user'](https://github.com/litewhat/fulcro-getting-started/pull/3/files#diff-ee6061193b3ed5d8d69e3d1891be2abdR60)) use `RETURNING` clause.